qp_qiime2/tests/test_qiime2.py

```diff
@@ -167,10 +172,11 @@ def test_pcoa(self):
 
         # non phylogenetic beta diversity
         params = {
-            'i-table': aid, 'p-metric': 'euclidean',
-            'i-tree': 'None'}
+            'BIOM table': aid, 'Diversity metric': 'Euclidean distance',
+            'Phylogenetic tree': 'None'}
```

Comment:
Is there an enforcement that absolutely ensures a phylogenetic metric will error appropriately if "Phylogenetic tree" is "None", and similarly, will it error appropriately if a tree is specified with a non-phylogenetic metric? I'm not seeing these tests but perhaps I'm missing them, would it be possible to add if they're not already included?